src/03_modules.py

```diff
@@ -5,27 +5,47 @@
 level operating system functionality.
 """
 
+# import os
+import os
 import sys
+
 # See docs for the sys module: https://docs.python.org/3.7/library/sys.html
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
 
+for i in sys.argv[0]:
+    print(i)
+
```

Comment:
What does this end up printing? `sys.argv` is a list and we want print out each element in the list